src/utils.rs

```diff
@@ -50,11 +50,19 @@ pub fn install_with_ledgerctl(
     dir: &std::path::Path,
     app_json: &std::path::Path,
 ) {
-    let out = Command::new("ledgerctl")
-        .current_dir(dir)
-        .args(["install", "-f", app_json.to_str().unwrap()])
-        .output()
-        .expect("fail");
+    let out = if cfg!(target_os = "macos") {
+        Command::new("python3")
+            .current_dir(dir)
+            .args(["-m", "ledgerctl", "install", "-f", app_json.to_str().unwrap()])
+            .output()
+            .expect("fail")
+    } else {
+        Command::new("ledgerctl")
+            .current_dir(dir)
+            .args(["install", "-f", app_json.to_str().unwrap()])
+            .output()
+            .expect("fail")
+    };
```

Comment:
This change seems unreliable. For older versions of `ledgerctl` it was working, but for newer ones it doesn't. I'm going to rollback it.